packages/api/routes/apikey/createKeyMiddleware.js

```diff
@@ -0,0 +1,12 @@
+const db_apikey = require("../../lib/db.apikey");
+
+// Return a function for creating an API key for a given user.
+module.exports = function createKeyMiddleware() {
+  return async (req, res, next) => {
+    const apikey = await db_apikey.attach();
+    apikey.createKey(req.body.user).then(doc => {
+      res.send(doc);
+      next();
+    });
```

Comment:
@sayak-sarkar Can you please use async/await for this part as well?

```suggestion
    const apikey = await db_apikey.attach();
    const doc = await apikey.createKey(req.body.user);
    res.send(doc);
    next();
```